tpl/corona_graph_header.tpl

```diff
@@ -1,7 +1,9 @@
-<span>
-    <img src="{{flag}}.png" class="flag">{{country}}: <i class="fas fa-hospital fx"
-        style="color:{{color}};">&nbsp;</i>{{confirmed}}&nbsp;
-    <i class="fas fa-skull-crossbones fx" style="color:{{color}};">&nbsp;</i>{{deaths}}&nbsp;
-    <i class="fas fa-users fx" style="color:{{color}};">&nbsp;</i>{{ratio}}
-    <i class="fas fa-angle-double-up fx" style="color:{{color}};">&nbsp;</i>{{doubling}}
+<span style="display:inline-flex;align-items: baseline;">
+    <img src="{{flag}}.png" class="flag" style="height: 13px;">{{country}}:&nbsp; 
+    <div style="color:white;display: {{showConfirmed}};"><i class="fas fa-hospital fx" style="color:{{color}};">&nbsp;</i>{{confirmed}}&nbsp;</div>
```

Comment:
Did you consider to conditionally render  the complete div instead of setting the display parameter?
like:` {{#if showConfirmed}} <div style="color:white;"> .... {{/if}}`

Reply:
Done. It's easy with fresh eyes.